x-pack/plugins/saved_objects_tagging/server/saved_objects/tagging_saved_objects_client_wrapper.ts

```diff
@@ -0,0 +1,171 @@
+  }
+
+  async find<T = unknown>(options: SavedObjectsFindOptions) {
+    return this.options.baseClient.find<T>(options);
+  }
+
+  async bulkGet<T = unknown>(
+    objects?: SavedObjectsBulkGetObject[],
+    options?: SavedObjectsBaseOptions
+  ) {
+    return this.options.baseClient.bulkGet<T>(objects, options);
+  }
+
+  async get<T = unknown>(type: string, id: string, options?: SavedObjectsBaseOptions) {
+    return this.options.baseClient.get<T>(type, id, options);
+  }
+
+  async resolve<T = unknown>(type: string, id: string, options?: SavedObjectsBaseOptions) {
+    return this.options.baseClient.resolve<T>(type, id, options);
+  }
+
+  async update<T = unknown>(
+    type: string,
+    id: string,
+    attributes: Partial<T>,
+    options?: SavedObjectsUpdateOptions
+  ) {
+    return this.options.baseClient.update<T>(type, id, attributes, options);
+  }
+
+  async addToNamespaces(
+    type: string,
+    id: string,
+    namespaces: string[],
+    options?: SavedObjectsAddToNamespacesOptions
+  ) {
+    return this.options.baseClient.addToNamespaces(type, id, namespaces, options);
+  }
+
+  async deleteFromNamespaces(
+    type: string,
+    id: string,
+    namespaces: string[],
+    options?: SavedObjectsDeleteFromNamespacesOptions
+  ) {
+    return this.options.baseClient.deleteFromNamespaces(type, id, namespaces, options);
+  }
+
+  async bulkUpdate<T = unknown>(
+    objects: Array<SavedObjectsBulkUpdateObject<T>>,
+    options?: SavedObjectsBulkUpdateOptions
+  ) {
+    return this.options.baseClient.bulkUpdate<T>(objects, options);
+  }
+
+  async removeReferencesTo(
+    type: string,
+    id: string,
+    options?: SavedObjectsRemoveReferencesToOptions
+  ) {
+    return this.options.baseClient.removeReferencesTo(type, id, options);
+  }
+
+  async openPointInTimeForType(
+    type: string | string[],
+    options?: SavedObjectsOpenPointInTimeOptions
+  ) {
+    return this.options.baseClient.openPointInTimeForType(type, options);
+  }
+
+  async closePointInTime(id: string, options?: SavedObjectsBaseOptions) {
+    return this.options.baseClient.closePointInTime(id, options);
+  }
+
+  createPointInTimeFinder(
+    findOptions: SavedObjectsCreatePointInTimeFinderOptions,
+    dependencies?: SavedObjectsCreatePointInTimeFinderDependencies
+  ): ISavedObjectsPointInTimeFinder {
+    return this.options.baseClient.createPointInTimeFinder(findOptions, dependencies);
+  }
+
+  async collectMultiNamespaceReferences(
+    objects: SavedObjectsCollectMultiNamespaceReferencesObject[],
+    options: SavedObjectsCollectMultiNamespaceReferencesOptions = {}
+  ) {
+    const { excludeTags } = options;
+    const typesToExclude = [
+      ...(options.typesToExclude ?? []),
+      ...(excludeTags ? [tagSavedObjectTypeName] : []),
+    ];
+    const newOptions = { ...options, ...(typesToExclude.length && { typesToExclude }) };
+    return this.options.baseClient.collectMultiNamespaceReferences(objects, newOptions);
+  }
```

Comment:
see my first comment

Reply:
I removed the tagging wrapper along with the `excludeTags` and `typesToExclude` options. Now, `collectMultiNamespaceReferences` just returns the entire object graph and the client is responsible for filtering any specific types out before sharing.